scripts/configure

```diff
@@ -26,98 +14,194 @@ check_dependencies () {
   done
 }
 
-check_dependencies wget docker docker-compose
+if [ ! $(uname -s) == "Linux" ]; then	
```

Comment:
uname command output should be quoted here, I'll do that in a PR along with a few other tweaks.